docs/en/operations/system-tables/dead_letter_queue.md

```diff
@@ -0,0 +1,71 @@
+---
+slug: /en/operations/system-tables/dead_letter_queue
+---
+# dead_letter_queue
+
+Contains information about messages received via a stream engine and parsed with an errors. Currently implemented for Kafka and RabbitMQ.
+
+Logging is controlled by `dead_letter_queue` of `kafka_handle_error_mode` setting.
+
+The flushing period of data is set in `flush_interval_milliseconds` parameter of the [dead_letter_queue](../../operations/server-configuration-parameters/settings.md#server_configuration_parameters-dead_letter_queue) server settings section. To force flushing, use the [SYSTEM FLUSH LOGS](../../sql-reference/statements/system.md#query_language-system-flush_logs) query.
+
+ClickHouse does not delete data from the table automatically. See [Introduction](../../operations/system-tables/index.md#system-tables-introduction) for more details.
+
+Columns:
+
+- `stream_type` ([Enum8](../../sql-reference/data-types/enum.md)) - Stream type. Possible values: 'Kafka'.
```

Comment:
```suggestion
- `stream_type` ([Enum8](../../sql-reference/data-types/enum.md)) - Stream type. Possible values: 'Kafka' or 'RabbitMQ'.
```

Reply:
In similar case in settings.md it is "and", not "or".
ChatGPT agrees that "and" is slightly better.

Reply:
Yeah, that is completely fine for me as long as it is mentioned. Thanks for checking it though!